CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,17 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.48.3...master)
+
+## FxA Client
+
+### Breaking Changes
+
+* `migrateFromSessionToken` now returns a metrics JSON object if the migration succeeded.
+  ([#2492](https://github.com/mozilla/application-services/pull/2492))
+
+### Features
+
+* `migrateFromSessionToken` now handles offline use cases. It caches the data the consumers originally provide.
+  If there's no network connectivity then the migration could be retried using the new `retryMigrateFromSessionToken` method. 
+  Consumers may also use the `isInMigrationState` method to check if there's a migration in progress.
+  ([#2492](https://github.com/mozilla/application-services/pull/2492))
```

Comment:
I suggest rewording this for clarity, to focus on the things that consumers need to know in the order they need to know them. Along the lines of:

* `migrateFromSessionToken` now has special handling for transient failures such as network errors. If the migration fails due to a transient error, then the provided credentials will be cached so that the migration can be retried later. Consumers can call the `isInMigrationState` method to check if there is a cached migration in progress, and `retryMigrateFromSessionToken` to retry it.